src/crimson/os/seastore/transaction_manager.cc

```diff
@@ -467,7 +467,7 @@ TransactionManager::rewrite_logical_extent(
       lextent->get_paddr(),
       nlextent->get_length(),
       nlextent->get_paddr(),
-      nlextent.get());
+      nlextent.get()).discard_result();
   } else {
     assert(get_extent_category(lextent->get_type()) == data_category_t::DATA);
     auto extents = cache->alloc_new_data_extents_by_type(
```

Comment:
I suppose data extents are special here that the shared mapping is allowed to be splitted without the need to adjust the indirect mappings.